src/samples/CopilotStudioClient/CopilotStudioClient/README.md

```diff
@@ -99,30 +110,39 @@ This step will require permissions to Create application identities in your Azur
 > [!TIP]
 > If you do not see `Power Platform API` in the list of API's your organization uses, you need to add the Power Platform API to your tenant. To do that, goto [Power Platform API Authentication](https://learn.microsoft.com/power-platform/admin/programmability-authentication-v2#step-2-configure-api-permissions) and follow the instructions on Step 2 to add the Power Platform Admin API to your Tenant
 
+> [!TIP]
+> If you see the following error message:
+> ```sh
+> fail: Microsoft.Agents.CopilotStudio.Client.CopilotClient[0]
+>      Error: {"code":"Forbidden","message":"The caller is not authorized to perform the request.","innererror":{"code":"InsufficientDelegatedPermissions","message":"Authorization denied: Application missing required delegated permissions: [CopilotStudio.Copilots.Invoke, All.All.ReadWrite]"}}
+> ```
+> You may also need to add the `Directory.ReadWrite.All` permission, found under `Microsoft APIs` -> `Microsoft Graph` -> `Delegated permissions` -> `Directory` -> `Directory.ReadWrite.All`. You will also need an admin to grant consent.
+
 ### Instructions - Configure the Example Application - Service Principal Login
 
 With the above information, you can now run the client `CopilostStudioClientSample`.
 
 1. Open the appSettings.json file for the CopilotStudioClientSample, or rename launchSettings.TEMPLATE.json to launchSettings.json.
-1. Configured the placeholder values for the various key's based on what was recorded during the setup phase.
+2. Configured the placeholder values for the various key's based on what was recorded during the setup phase.
```

Comment:
Unless you explicitly need this to be item 2 in the list, I'd suggest using the md syntax of a 1 in front of every item and letting the renderer do the numbering.